samples/ios/app/sdk_generator.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Fail the script on first error
+set -e
```

Comment:
eugh :o Do we *really* need to copy this over? Can't we symlink or something?

Reply:
For now ... we kinda do, unless I add some of the parameters to the xcode invocation, let me try that.

Reply:
If you prefer, we can have the sample app changes as a separate PR and can move forward landing the first part, if others agree

Reply:
Nope, let's do this right.